src/content/docs/configuration/providers.mdx

```diff
@@ -10,7 +10,7 @@ import Aside from '@components/Aside.astro'
 Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
 
 1. **Provider**  
-    A plugin that interfaces with the [Daytona Server](../reference/server), responsible for Workspace deployment and lifecycle management.
+    A plugin that interfaces with the [Daytona Server](/usage/server), responsible for Workspace deployment and lifecycle management.
```

Comment:
@Tpuljak I have already changed and tested. It's already working. See video above.